src/replay/confirm_slot.zig

```diff
@@ -2,6 +2,7 @@ const std = @import("std");
 const sig = @import("../sig.zig");
 const replay = @import("lib.zig");
 const tracy = @import("tracy");
+const vote_listener = @import("../consensus/vote_listener.zig");
```

Comment:
I don't personally have a strong opinion on this matter, but I think the current consensus on imports like this is to alias them relative to `sig`, in this case `sig.consensus.vote_listener`.

Reply:
[8055677](https://github.com/Syndica/sig/pull/875/commits/8055677251a5c40de3fb83c40ff31149c696195e)